src/test/DoS_tests.cpp

```diff
@@ -23,7 +23,7 @@
 #include <boost/foreach.hpp>
 #include <boost/test/unit_test.hpp>
 
-// Tests this internal-to-main.cpp method:
+// Tests this internal-to-net_processing.cpp method:
```

Comment:
May as well correct the entire comment while you're doing this:

`// Tests these internal net_processing.cpp methods:`